src/profile-logic/profile-data.js

```diff
@@ -253,6 +253,23 @@ export function getTimingsForPath(
   };
   let rootTime = 0;
 
+  function accumulateToTimings(
+    timings: {
+      breakdownByImplementation: BreakdownByImplementation | null,
+      value: number,
+    },
+    stackIndex: IndexIntoStackTable,
+    funcIndex: IndexIntoFuncTable
+  ): void {
+    timings.value += interval;
+
+    let implementation = 'native';
```

Comment:
This a small optional observation, but I think would be easier to read as a ternary, and avoids a `let` reassignment.

Reply:
Right, it wasn't as simple before I extracted getJsImplementationForStack, but now a ternary would be just perfect.